dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task04220RemoveDeleteInactiveClusterServersJob.java

```diff
@@ -0,0 +1,64 @@
+package com.dotmarketing.startup.runonce;
+
+import java.util.List;
+
+import com.dotcms.repackage.org.apache.commons.lang.StringUtils;
+import com.dotcms.util.CollectionsUtils;
+import com.dotmarketing.startup.AbstractJDBCStartupTask;
+
+/**
+ * This task removes the database information associated to the
+ * {@code com.dotmarketing.quartz.job.DeleteInactiveClusterServersJob} class that was deleted as of
+ * dotCMS 4.2. This will make sure that upgrading environments will not be trying to start a Quartz
+ * Job that doesn't exist anymore.
+ * 
+ * @author Jose Castro
+ * @version 4.2
+ * @since Sep 28, 2017
+ *
+ */
+public class Task04220RemoveDeleteInactiveClusterServersJob extends AbstractJDBCStartupTask {
+
+    // Execution order of the queries is important
+    private static final List<String> QUERIES = CollectionsUtils.list(
+                    "DELETE FROM qrtz_excl_fired_triggers WHERE trigger_name = 'trigger23'",
+                    "DELETE FROM dotcms.qrtz_excl_cron_triggers WHERE trigger_name = 'trigger23'",
+                    "DELETE FROM dotcms.qrtz_excl_triggers WHERE trigger_name = 'trigger23' AND job_name = 'RemoveInactiveClusterServerJob'",
+                    "DELETE FROM dotcms.qrtz_excl_job_details WHERE job_name = 'RemoveInactiveClusterServerJob' AND job_class_name = 'com.dotmarketing.quartz.job.DeleteInactiveClusterServersJob'");
+
+    @Override
+    public boolean forceRun() {
+        return Boolean.TRUE;
+    }
+
+    @Override
+    public String getPostgresScript() {
+        return StringUtils.join(QUERIES, ";");
+    }
+
+    @Override
+    public String getMySQLScript() {
+        return StringUtils.join(QUERIES, ";");
+    }
+
+    @Override
+    public String getOracleScript() {
+        return StringUtils.join(QUERIES, ";");
+    }
+
+    @Override
+    public String getMSSQLScript() {
+        return StringUtils.join(QUERIES, ";");
+    }
+
+    @Override
+    public String getH2Script() {
+        return null;
+    }
+
+    @Override
+    protected List<String> getTablesToDropConstraints() {
+        return null;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Return an empty collection instead of null. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1168)